Python/PRP/GUI/pyGUIControlMod.h

```diff
@@ -0,0 +1,34 @@
+/* This file is part of HSPlasma.
+*
+* HSPlasma is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* HSPlasma is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with HSPlasma.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef _PYGUICONTROLMOD_H
+#define _PYGUICONTROLMOD_H
+
+#include "PyPlasma.h"
+#include <PRP/GUI/pfGUIControlMod.h>
+
+PY_WRAP_PLASMA_VALUE(GUIColorScheme, class pfGUIColorScheme)
+PY_WRAP_PLASMA(GUIControlMod, class pfGUIControlMod)
+
+
+/* Python property helpers */
+inline PyObject* pyPlasma_convert(const pfGUIColorScheme& value) { return pyGUIColorScheme_FromGUIColorScheme(value); }
+
+template <> inline int pyPlasma_check<pfGUIColorScheme>(PyObject *value) { return pyGUIColorScheme_Check(value); }
+
+template <> inline pfGUIColorScheme pyPlasma_get(PyObject* value) { return *((pyGUIColorScheme*)value)->fThis; }
```

Comment:
The `pyPlasma_convert` and `pyPlasma_get` overloads should take and return an address so they work both with `PY_PROPERTY_PROXY` (see comments below) and pointer properties (for GUIControlMod).